core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala

```diff
@@ -366,7 +366,7 @@ class HistoryServerSuite extends SparkFunSuite with BeforeAndAfter with Matchers
     server.attachHandler(contextHandler)
 
     implicit val webDriver: WebDriver =
-      new HtmlUnitDriver(BrowserVersion.INTERNET_EXPLORER_11, true)
+      new HtmlUnitDriver(BrowserVersion.INTERNET_EXPLORER, true)
```

Comment:
Just wondering why Internet Explorer at all. Why no `BrowserVersion.getDefault()`?

Reply:
https://github.com/apache/spark/pull/24843#discussion_r294049535 is relevant, but, I imagine any modern browser would handle the jquery JS fine too. I can try just removing this specification, as it (now) defaults to Chrome (51). I think before it emulated IE 8.